src/components/WebLogView/WebLogView.tsx

```diff
@@ -41,43 +43,36 @@ export const WebLogView = memo(function WebLogView({
     onSelectRequest(selectedRequest)
   }, [selectedRequest, onSelectRequest])
 
-  if (groups !== undefined) {
```

Comment:
This condition was added to support both grouped and ungrouped recordings, but we always default to at least one group (even for imported hars) so this condition became redundant.